include/k52/optimization/hooke_jeeves_method.h

```diff
@@ -0,0 +1,76 @@
+#ifndef HOOKE_JEEVES_METHOD_H_
+#define HOOKE_JEEVES_METHOD_H_
+
+#include <boost/shared_ptr.hpp>
+
+#include <k52/optimization/continuous_optimizer.h>
+
+namespace k52
+{
+namespace optimization
+{
+
+/// @class HookeJeevesMethod
+/// @brief Realization of Hooke-Jeeves optimization method.
+class HookeJeevesMethod: public ContinuousOptimizer
+{
+public:
+	/// @brief Creates instance of HookeJeevesMethod.
+	/// @param init_step - number that defines value of step on first iteration
+	/// @param max_iteration_number - maximum number of iterations before computations will be stopped
+	/// @param precision - number that defines accuracy of our computations
+	HookeJeevesMethod(
+		double acceleration = 3.5,
```

Comment:
Missing comment with detailed description of parameter with explanation why default value is better